runtime/src/main/java/org/teiid/odbc/ODBCServerRemoteImpl.java

```diff
@@ -913,6 +913,10 @@ else if (sql.equals("SELECT typinput='array_in'::regproc, typtype FROM pg_catalo
             if (modified.startsWith(PK_QUERY)) {
                 return PK_REPLACEMENT_QUERY + modified.substring(PK_QUERY.length());
             }
+
+            if (sql.equalsIgnoreCase("select version()")){ //$NON-NLS-1$
+                return "select version() as version"; //$NON-NLS-1$
```

Comment:
I'd prefer not to have these kind of special purpose workarounds.  It could be an option for the resolver to auto alias in a similar manner as pg.

Reply:
is that a feature we need to add or already available?

Reply:
It would need to be added.  It would be a relatively small change to the auto-naming that we do to produce the exprN names.

Reply:
Ok,let's reject this PR and I will assign the JIRA to you.  I will test once you have a fix. Can we get this in 13.1 pls